shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -77,6 +99,11 @@ @implementation FlutterViewController {
   // UIScrollView with height zero and a content offset so we can get those events. See also:
   // https://github.com/flutter/flutter/issues/35050
   fml::scoped_nsobject<UIScrollView> _scrollView;
+#ifdef __IPHONE_13_4
```

Comment:
These don't have to be ifdef'd.  I think it's better to limit the amount of conditional compilation.